src/components/List/List.story.jsx

```diff
@@ -64,7 +69,7 @@ export const sampleHierarchy = {
       'New York Mets': {
         'Jeff McNeil': '3B',
         'Amed Rosario': 'SS',
-        'Michael ConfortoMichael ConfortoMichael ConfortoMichael Conforto': 'RF',
```

Comment:
I think this was originally here to make sure long list items render correctly. Could you leave it so we can quickly check list items in the future? It does look like a typo though so maybe leave a comment on it?

Reply:
Lets rename to " A long list title to showcase what happens when titles is too long for the container...yada yada" or whatever. This way it doesn't look accidental.

Reply:
Will put this back, it looked like a visual bug at first.